src/firefly/js/visualize/HiPSCntlr.js

```diff
@@ -0,0 +1,193 @@
+import {flux} from '../Firefly.js';
```

Comment:
This is good.  I had not considered keeping the hips list in the store.  This is good.